src/Firehose.Web/Firehose.Web.csproj

```diff
@@ -196,7 +196,6 @@
     <Compile Include="App_Start\RouteConfig.cs" />
     <Compile Include="Authors\AllanRitchie.cs" />
     <Compile Include="Authors\AndrewHoefling.cs" />
-    <Compile Include="Authors\BenceLenart.cs" />
```

Comment:
I just tried the feed and it seems to return data. I did however notice that the uri for the website url is missing the "h" in http.

Reply:
I guess he has fixed it. When I fetched the feed during my test it didn't contain any items.